.github/workflows/build.yaml

```diff
@@ -123,18 +123,11 @@ jobs:
         run: |
           mkdir ${{ env.RELEASE_PATH }}
           cp ./dist/${{ matrix.target_os}}_${{ matrix.target_arch}}/release/rad.exe ${{ env.RELEASE_PATH }}/rad_${{ matrix.target_os}}_${{ matrix.target_arch}}.exe
-      - name: Upload Release binaries 
-        uses: actions/upload-artifact@v3
-        with:
-          name: rad_cli_release
-          path: ${{ env.RELEASE_PATH }}
       - name: Upload CLI binary
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: rad_cli_${{ matrix.target_os}}_${{ matrix.target_arch}}
-          path: |
-            ./dist/${{ matrix.target_os}}_${{ matrix.target_arch}}/release/rad
-            ./dist/${{ matrix.target_os}}_${{ matrix.target_arch}}/release/rad.exe
```

Comment:
Is there any way to validate these changes?

Reply:
We uploaded cli binary twice. So i deleted one and also upload action v4 does not support append operation anymore. in v4, we need to upload separately and then download action needs to download all artifact with the pattern.

Reply:
Yes I validated it.

Reply:
I validated this by changing the workflow to list files. - https://github.com/radius-project/radius/actions/runs/8486662436/job/23253525031

<img width="757" alt="image" src="https://github.com/radius-project/radius/assets/11863108/18131bba-0eb4-4ba8-9e7e-1e3c02763c4c">